tests/GenerateScriptTests/GenerateScriptTests.Tests.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Infrastructure wise looks good aside from one last nit from my side, please also rename the project file to `GenerateScript.Tests.csproj`.